### ISSUE TITLE
How to get the thread's local storage address in a program's child thread?

### ISSUE THREAD

**Issue by MRLEE123456** [S2E/s2e-env#319]
Hi,Tanks for your help!
Do you have a way to get the thread's local storage address in a program's child thread?

**Comment by MRLEE123456** [comment]
Is it convenient to provide an OSMonitor.cpp that contains this api?

**Comment by vitalych** [comment]
Yes. You will need an abstract method in OSMonitor.h and implement it for each type of OS. Windows/Linux manage TLS differently.
